frontend/Icons/Icons/magic_button_FILL0_wght100_GRAD0_opsz24.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" width="24"><path d="m10.4 17.6-2-4.4-4.4-2 4.4-2 2-4.4 2 4.4 4.4 2-4.4 2Zm6.4 1.6-1-2.2-2.2-1 2.2-1 1-2.2 1 2.2 2.2 1-2.2 1Z"/></svg>
```

Comment:
Are we using this file anywhere ? If yes, I don't this is the right location to put it